src/docs/services/email.mdx

```diff
@@ -21,6 +19,8 @@ The backend to be used for email delivery. Options are `smtp`, `console`, and `d
 
 Defaults to `smtp`. Use `dummy` if you’d like to disable email delivery.
 
+Use `sentry.utils.email.PreviewBackend` to receive mail locally while developing. By default they'll open in the native Mail app if using a Macbook.
```

Comment:
I'm on the fence about adding a mention of having to select iCloud as the provider and sign in with my work email (and specifying this is for Sentry staff). It seems really specific but also is what I was having a hard time with.